cdk/kittyhawk/.eslintrc.json

```diff
@@ -12,7 +12,7 @@
   "parserOptions": {
```

Comment:
Also as a general question, is there a reason you create new constructs, like `class Certificate extends Construct` instead of extending the existing Certificate object? I do it the other way in kraken but I've got no idea which one would be considered a "better" option. One reason I think my approach may be better is that you only have constructs that are in the final synthesized construct tree/chart, but I don't know if that's important

Reply:
For containers, could you explain a bit about why you went with a class that implements the ContainerInterface? I have a very similar situation with Steps in cdkactions and I'm honestly not sure what the best way of representing them are.

Reply:
Still not sure how I feel about the `synth` chart shortcut. Let's talk a bit more about it